gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -62,6 +62,24 @@
  */
 #define GST_TENSOR_TYPE_ALL "{ float32, float64, int64, uint64, int32, uint32, int16, uint16, int8, uint8 }"
 
+/**
+ * @brief Possible tensor formats
+ */
+#define GST_TENSOR_FORMAT_ALL "{ static, flexible, sparse }"
+
+/**
+ * @brief Caps string for the caps template of tensor stream.
+ * format should be a string that describes the data format, or possible formats of incoming tensor.
+ *
+ * If the data format is static, another tensor information should be described in caps.
+ * - num-tensors: The number of tensors in a GstBuffer.
+ * - types: The list of data type in the tensor. type should be one of types in GST_TENSOR_TYPE_ALL. (types=(string)"unit8,int32,flat32")
+ * - dimensions: The list of dimension in the tensor. dimension should be a formatted string with rank NNS_TENSOR_RANK_LIMIT. (dimensions=(string)"dim1:dim2:dim3:dim4,dim1:dim2:dim3:dim4")
+ */
+#define GST_TENSOR_CAPS_MAKE(fmt) \
+    NNS_MIMETYPE_TENSOR "," \
```

Comment:
not `NNS_MIMETYPE_TENSORS`?
From https://github.com/nnstreamer/nnstreamer/issues/3351, `other/tensor` will be deprecated.

But I think `other/tensor,format=static,num-tensors=3,types=....` looks better.

Reply:
Can you check the mimetype in tizen tct?
I think all testcases with appsrc would be other/tensor :)

Reply:
We are deprecating ```other/tensor``` soon. Thus, we are not going to add ```format=....``` to ```other/tensor```.
The cap syntax for ```other/tensor``` will not be updated.

```format=...``` will be added to ```other/tensors``` only.

Reply:
Refer to https://lists.lfaidata.foundation/g/nnstreamer-technical-discuss/topic/new_tensor_stream_types/82277609?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,82277609

Reply:
`other/tensor` vs `other/tensors`

Reply:
We are deprecating ```other/tensor``` and this is supported for backward-compatibility only; thus, we do not need to add additional semantics to ```other/tensor```.

Reply:
updated